website/components/layouts/StandardLayout.tsx

```diff
@@ -9,17 +9,28 @@ import Menu from "components/partials/Menu"
 type Props = {
   noCta?: boolean
   withMenu?: boolean
-}
+} & MetaTags
```

Comment:
To avoid adding `& MetaTags` to each Props you can create a wrapper around `React.FC` that adds it. Something like:
LayoutComponent<P> = React.FC<P & MetaTags>